volume/volume.go

```diff
@@ -104,8 +110,9 @@ type MountPoint struct {
 	CopyData bool `json:"-"`
 	// ID is the opaque ID used to pass to the volume driver.
 	// This should be set by calls to `Mount` and unset by calls to `Unmount`
-	ID   string `json:",omitempty"`
-	Spec mounttypes.Mount
+	ID    string `json:",omitempty"`
+	Tmpfs *Tmpfs `json:"-"`
```

Comment:
Unlike the spec structure, `*Tmpfs` here contains `RawOptions` string, which is needed to the support for the old tmpfs in `docker run`.